script.js

```diff
@@ -35,22 +35,22 @@ function startVideo() {
 startVideo();
 
 video.addEventListener('play', () => {
-  
+
   setInterval(async () => {
-    const detections = await faceapi.detectAllFaces(video, 
+    const detections = await faceapi.detectAllFaces(video,
       new faceapi.TinyFaceDetectorOptions())
       .withFaceExpressions()
-    if (detections[0].expressions.happy > 0.3)
+    if (detections && detections[0] && detections[0].expressions.happy > 0.3)
```

Comment:
can expressions and happy property be nil as well ?

Reply:
I am not sure but I think they'll carry 0 as a value at minimum.